.github/workflows/main.yml

```diff
@@ -0,0 +1,65 @@
+name: CI/CD
+
+on:
+  push:
+    branches: [INV-26/deploy-to-vps]
+
+jobs:
+    build-and-test:
+      runs-on: ubuntu-latest
+      strategy:
+          matrix:
+              node-version: [20.x]
+
+      steps:
+          - name: Git checkout
+            uses: actions/checkout@v4
+
+          - name: Setup Node.js
+            uses: actions/setup-node@v4
+            with:
+                node-version: ${{ matrix.node-version }}
+
+          - name: Install dependencies
+            run: npm ci
+
+          - name: Lint code
+            run: npm run lint
+          
+          - name: Build the project
+            run: npm run build
+
+          - name: Set up Docker Buildx
+            uses: docker/setup-buildx-action@v3
+
+          - name: Log in to Docker Hub
+            uses: docker/login-action@v2
+            with:
+              username: ${{ secrets.DOCKER_USERNAME}}
+              password: ${{ secrets.DOCKER_PASSWORD }}
+
+          - name: Build Docker image
+            run: |
+              docker build \
+                --file Dockerfile \
+                --tag ${{ secrets.DOCKER_USERNAME}}/invention-airbnb:1.0.0 .
```

Comment:
Let's add the tag to the environment. So it'll be easier to modify in the future.